src/hooks/Guilds/useVoteSummary.ts

```diff
@@ -8,15 +8,16 @@ export default function useVoteSummary(
   guildId: string,
   proposalId: string
 ): number[] {
-  const { data: { totalVotes = null } = {} } = useProposal(guildId, proposalId);
+  const { data: { totalVotes = null, totalActions } = {} } = useProposal(

```

Comment:
In the new optimised contracts we dont have totalActions available anymore, we need to use totalVotes.length-1

Reply:
Sure. done